src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -189,6 +189,39 @@
     type: command
     short-summary: List policy definitions.
 """
+helps['policy setdefinition create'] = """
+            type: command
+            short-summary: Create a policy set definition.
+            parameters:
+                - name: --definitions
+                  type: string
+                  short-summary: Policy definitions in JSON format, or a path to a file containing JSON rules.
+            examples:
+                - name: Create a policy set definition.
+                  text: |
+                    az policy setdefinition create -n readOnlyStorage --definitions \\
+                        [ \\
+                            { \\
+                                "policyDefinitionId": "/subscriptions/mySubId/providers/Microsoft.Authorization/policyDefinitions/storagePolicy" \\
```

Comment:
Not just IDs, it can have the entire policy rule as json as well.

Reply:
Got it

Reply:
I'm sorry I misspoke above. The policy set cannot contain entire policy rule, but it can have a multiples of policyDefinitionId and parameters (which is json object), so either way, this is a complex object that requires file handling